### PR TITLE
feat(devserver): Force latest offset in devserver

### PR DESCRIPTION
**Pull request by mitsuhiko**
Currently if you don't run the devservices continuously before the topic gets too old you will have a stuck snuba. Right now the workflow is to recreate the underlying volumes but that seems pretty silly.

There does not seem to be much of a disadvantage to forcing the offset to latest. In fact this might even make it a more sensible setup as you will not have to chug through old alerts and similar nonsense in dev.

Requires https://github.com/getsentry/arroyo/pull/54

### CODE REVIEW

File: Makefile
Comment (mitsuhiko) [inline Makefile:27]: TODO: remove this and bump arroyo in requirements instead.

File: Makefile
Comment (mitsuhiko) [inline Makefile:27]: This needs undoing before merge with a bump to a new arroyo release.

Comment (rahul-kumar-saini) [reply Makefile:29]: This should unblock:
https://github.com/getsentry/snuba/pull/2670

File: snuba/consumers/strict_consumer.py
Comment (lynnagara) [inline snuba/consumers/strict_consumer.py:69]: Didn't this end up being `arroyo.strict.offset.reset`?

File: snuba/subscriptions/consumer.py
Comment (lynnagara) [inline snuba/subscriptions/consumer.py:197]: You can remove force now

**Review by lynnagara — APPROVED** [review]
Thanks, will make a lot of people happy

**Pull request merged**
